examples/message/SlackMessageExample.js

```diff
@@ -0,0 +1,94 @@
+const { Reshuffle } = require('reshuffle')
+const { SlackConnector, SlackEventType, SlackEvents } = require('reshuffle-slack-connector')
+const app = new Reshuffle()
+
+const main = async () => {
+  const slackConnector = new SlackConnector(app, {
+    token: '<your_slack_token_starting_with_xox>',
+    signingSecret: '<your_slack_signing_secret>',
+    port: '<slack_receiver_port>', // Default to 3000
+    endpoints: '<slack_receiver_endpoints>', // Default to '/'
+  })
+
+  // Events
+  slackConnector.on(
+    {
+      type: SlackEventType.EVENT,
```

Comment:
SlackEventType.EVENT sounds confusing

Reply:
I kept the same semantic as runtime v1. I don't have a better name to suggest but I think it makes sense when you look at the other types